dev/bots/prepare_package.dart

```diff
@@ -83,15 +99,27 @@ class ArchiveCreator {
     runGit(<String>['remote', 'add', 'origin', GITHUB_REPO]);
   }
 
+  /// Retrieve the MinGit executable from storage and unpack it.
+  Future<Null> installMinGit() async {
```

Comment:
maybe installMinGitForWindows to avoid confusion in the future?

Reply:
I changed it to installMinGitIfNeeded and if it's not windows, it just returns.